index.html

```diff
@@ -1,480 +1,225 @@
 <!DOCTYPE html>
-
-<html>
-
-  <head>
-
-    <meta charset="UTF-8">
-
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <title>Johnson's Portfolio</title>
-
-    <link rel="stylesheet" href="styles.css">
-
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-
-    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Montserrat:wght@400;700&family=Playfair+Display:wght@400;500&family=Poppins:wght@100;300;400&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
-  
-  </head>
-
-  <body>
-
-    <header>
-      <nav>
-        <ul>
-          <li>
-            <a href="#homepage">Home</a>
-          </li>
-          <li>
-            <a href="#">Portfolio</a>
-          </li>
-          <li>
-            <a href="#">About</a>
-          </li>
-          <li>
-            <a href="#">Contact</a>
-          </li>
-        </ul>
-        <a class="menu-burger" href="#"><img src="icons/ic_menu.png"></a>
-      </nav>
-    </header>
-
-
-    <main>
-      <!-- start home page section  -->
-      <section class="homepage">
-
-        <div class="container">
-
-          <h1>
-
-            Johnson Macharia
-
-          </h1>
-
-          <p>
-            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br><br>
-            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Johnson Macharia</title>
+  <link href="./styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;600;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="#homepage">Home</a></li>
+        <li><a href="#projects-container">Portfolio</a></li>
+        <li><a href="#about-me-container">About</a></li>
+        <li><a href="#contact-form">Contact</a></li>
+      </ul>
+      <a class="menu" href="#"><img src="icons/ic_menu.png"></a>
+    </nav>
+
+    <div id="mobile_menu">
+
+      <a><img id="x-btn" src="icons/Union1.png"></a>
+
+      <ul class="menu-bar">
+        <li><a href="#homepage">Hello</a></li>
+        <li><a href="#projects-container">Portfolio</a></li>
+        <li><a href="#about-me-container">About</a></li>
+        <li><a href="#contact-form">Contact</a></li>
```

Comment:
👏👏 Excellent work on implementing the mobile menu functionality! The navigation is working partially. However, there are a couple of minor improvements needed:

- [ ] As per the project requirement, when the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear or close. Currently, the mobile menu does not close upon user selection. Please update your code to hide the mobile menu when a menu option is clicked.

- [ ] Additionally, when trying to click the nav items, sometimes it requires clicking on the same item twice to work. Please update your code so that these nav items work with just one click.

[Johnson-Macharia (1).webm](https://github.com/JohnteFresh/Portfolio-Development/assets/55840999/f1b06979-864e-4b52-954f-c056a2213e27)

By making these adjustments, your mobile menu will provide a smoother and more intuitive user experience. Keep up the great work, and happy coding! 🚀💻🌟

Reply:
I have updated the code.